view/chart/worker.ts

```diff
@@ -0,0 +1,100 @@
+import type { RenderType } from '../../stores/benchmark'
```

Comment:
```suggestion
import type { RenderType } from '../../stores/benchmark.js'
```

На конце импортов по правилам ESM всегда должно быть расширение. Ниже так же исправь (и проверь остальные импорты).